ansible/roles/grafana/handlers/main.yml

```diff
@@ -21,11 +21,11 @@
     method: POST
     body: "{{ item }}"
     body_format: json
-    status_code: 200,500
+    status_code: 200,500,409
```

Comment:
Curious - why is this needed? HTTP 409 seems to be something to do with conflicts; it would be good to have an understanding of why you added this.

Reply:
Added 409 to avoid task failing when the datasource already exist in Grafana.